### ISSUE TITLE
Ternary expression inside one liner function produces wrong output

### ISSUE THREAD

**Issue by netei** [eventualbuddha/decaffeinate#89]
Thanks for correcting all other issues :-) 

Input : 

`f=()->if a? then 1 else 0`

Outputs: 

`var f=function(){(typeof a !== "undefined" && a !== null) ? return 1 : return 0 };` 

Expected: 

```
var f=function(){return (typeof a !== "undefined" && a !== null) ? 1 : 0 };
```

The outputed version is not valid es6

**Comment by eventualbuddha** [comment]
[REPL](http://eventualbuddha.github.io/decaffeinate/repl/#?evaluate=true&code=f%3D()-%3Eif%20a%3F%20then%201%20else%200) shows the same output.

**Comment by eventualbuddha** [comment]
Fixed in v1.41.1.
